.scalafmt.conf

```diff
@@ -1,2 +1,14 @@
 maxColumn = 120
-align = none
\ No newline at end of file
+align = most
```

Comment:
I'm in two minds on this one. I do think it makes some things clearer, but sometimes it makes things look related when they're not. It also significantly adds to the size of this PR.

Reply:
Can you provide an example? We can tweak that parameter [in many ways](https://scalameta.org/scalafmt/docs/configuration.html#alignment). I agree, in general formatting will always increase the size of a PR, I think this is an exceptional case though -- in a "stable" branch the impact will be minor.

Reply:
Looking through again, none of them are that heinous.

One thing I really don't like on second viewing is moving multiline argument lists so that they start next to the opening parenthesis, rather than on s slightly indented new line. It moves a lot of information a long way from left alignment.

Reply:
Agreed, I've updated.